src/index.html

```diff
@@ -7,12 +7,68 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/style.css"
```

Comment:
Ensure that the SCSS file has been compiled into this CSS file. If the SCSS file is not compiled, the styles will not be applied correctly.